Dockerfile

```diff
@@ -1,4 +1,18 @@
-FROM ruby:2.6.6-stretch
+FROM ruby:2.6.6-alpine as build-stage
+
+ENV RAILS_ENV=production
+ENV BUNDLE_APP_CONFIG="app/.bundle"
+
+WORKDIR /app
+
+RUN apk add git build-base tzdata postgresql-dev
+
+COPY . /app
+RUN gem update bundler --no-document
+RUN bundle install --without="development test"  --jobs 4 --retry 3
```

Comment:
maybe if we also append:
```
 && rm -rf /usr/local/bundle/cache/*.gem \
     && find /usr/local/bundle/gems/ -name "*.c" -delete \
     && find /usr/local/bundle/gems/ -name "*.o" -delete
```
if will shave off a bit from the final size of the image

Reply:
Yes, it does shave 19MB irc.  It's still about 10% of the new size, so worth taking out.
I had `--no-cache` initially but I must have lost it in a rebase and it was causing deprecation messages as well.
I'll reintroduce a way to remove this bloat again.